packages/explorer/src/app/(explorer)/[chainName]/worlds/[worldAddress]/interact/Form.tsx

```diff
@@ -14,7 +15,7 @@ import { FunctionField } from "./FunctionField";
 export function Form() {
   const [hash] = useHashState();
   const { data, isFetched } = useWorldAbiQuery();
-  const [filterValue, setFilterValue] = useState("");
+  const [filterValue, setFilterValue] = useQueryState("functions", { defaultValue: "" });
```

Comment:
thinking now it should be just "function" because we're often looking for a specific function
```suggestion
  const [filterValue, setFilterValue] = useQueryState("function", { defaultValue: "" });
```

Reply:
ah true, updated